web/api/src/Service/StationService.ts

```diff
@@ -0,0 +1,60 @@
+import { StationModel } from '@/Model/StationModel'
+import logger from '@/loaders/logger'
+import { Station, StationType } from '@/lib/station/Station'
+import { v4 as UUID } from 'uuid'
+
+/**
+ * Station serivce performes the business logic between Grapgql/API and Database.
+ */
+export class StationService {
+    private _model: StationModel
+
+    constructor(model: StationModel) {
+        this._model = model
+    }
+
+    /**
+     * Returns a station, or null if a station is not found
+     * @param id id of the station to get
+     */
+    async getStationById(id: string) {
+        try {
+            let q = (await this._model.getStationById(id)).rows[0]
+        } catch (error) {
```

Comment:
antar den skal tilbake?
```suggestion
        try {
            let q = (await this._model.getStationById(id)).rows[0]
            return q
        } catch (error) {
```